src/operators/IntegralCurve/IntegralCurve.code

```diff
@@ -449,3 +478,60 @@ IntegralCurveViewerEnginePluginInfo::GetOperatorVarDescription(AttributeSubject
 
     return var;
 }
+
+Target: xml2atts
```

Comment:
The new logic looks good, Mark, thank you for adding it.

Reply:
@biagas thanks ❤️. I figured out, it only ever generates a single warning upon reading the value. So, after that first line in the python output above where I just reference `ica.MasterSlave`, and the warning appears, thereafter, the warning will not appear. I played with `PyExc_PendingDeprecationWarning` instead of just `NULL` in the coorresponding `PyErr_WarnFormat` call but that won't generate *visible* warnings unless the user takes action to show them.

I'll continue with same changes to other two classes that were impacted.